gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/InsertAllRequest.java

```diff
@@ -0,0 +1,399 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.bigquery;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Google Cloud BigQuery insert all request. This class can be used to stream data into BigQuery one
+ * record at a time without needing to run a load job. This approach enables querying data without
+ * the delay of running a load job. There are several important trade-offs to consider before
+ * choosing an approach.
+ *
+ * @see <a href="https://cloud.google.com/bigquery/streaming-data-into-bigquery">Streaming Data into
+ *     BigQuery</a>
+ */
+public class InsertAllRequest implements Serializable {
+
+  private static final long serialVersionUID = 211200307773853078L;
+
+  private final TableId table;
+  private final List<RowToInsert> rows;
+  private final Boolean skipInvalidRows;
+  private final Boolean ignoreUnknownValues;
+
+  /**
+   * A Google Big Query row to be inserted into a table. Each {@code RowToInsert} has an associated
+   * id used by BigQuery to detect duplicate insertion requests on a best-effort basis.
+   *
```

Comment:
You are right! I am not sure why my IDE complaint about it. I guess it was configured with Sun style (which seems to avoid the [empty line](http://www.oracle.com/technetwork/articles/java/index-137868.html))?

Also, when looking at guava code it seems that `<p>` is indeed preceded by an empty line however never after (the paragraph starts on the same line).